docs/src/api/class-playwright.md

```diff
@@ -135,6 +153,25 @@ with sync_playwright() as playwright:
 
 Returns a dictionary of devices to be used with [`method: Browser.newContext`] or [`method: Browser.newPage`].
 
+```csharp
+using Microsoft.Playwright;
+using System.Threading.Tasks;
+
+class PlaywrightExample
+{
+    public static async Task Main()
+    {
+        using var playwright = await Playwright.CreateAsync();
+        await using var browser = await playwright.Webkit.LaunchAsync();
+        await using IBrowserContext context = await browser.NewContextAsync(Playwright.Devices["iPhone 6"]);
```

Comment:
can it be simplified to `await using var context = ` similar to the above declarations?